scripts/deploy/aws.sh

```diff
@@ -91,6 +91,7 @@ function x1_terraform_args() {
     -var jupyterhub_extra_resource_limits="${JUPYTERHUB_EXTRA_RESOURCE_LIMITS}"
     -var gpu_enabled="${GPU_ENABLED}"
     -var gpu_type="${GPU_TYPE}"
+    -var deployment_type="aws"
```

Comment:
I think it may be better to introduce something like `nvidia_gpu_operator_enabled` instead, if we need provider-specific behavior. This could be passed from each provider's top-level script. So it will keep modules provider-agnostic.

Reply:
Agreed, this variable has been changed to enable_nvidia_operator. Let me know if AWS deployments with and without GPU work with the new change.